tests/corobo_test_case.py

```diff
@@ -0,0 +1,36 @@
+from errbot.backends.test import FullStackTest
+from errbot.plugin_info import PluginInfo
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
+
+import logging
+
+
+class CoroboTestCase(FullStackTest):
+
+    def setUp(self, plug_filename):
```

Comment:
Replace the parameter with plugin class

Reply:
Or a `plugins` set or a list, an iterable, for loading of plugins.

Reply:
I do not understand this one clearly. What is the purpose of this since we need only one plugin class at once?
>Or a plugins set or a list, an iterable, for loading of plugins.

Reply:
If we are testing dependent plugins? We need to mock more than one plugins, no?

Reply:
Yes right. Got it :+1: